src/main/java/com/faforever/api/config/security/WebSecurityConfig.java

```diff
@@ -48,7 +48,6 @@ public AuthenticationManager authenticationManagerBean() throws Exception {
   protected void configure(HttpSecurity http) throws Exception {
     // @formatter:off
       http
-      .csrf().disable() // http://stackoverflow.com/a/29917946
```

Comment:
👎 

If you remove this, `PATCH` and `POST` request from the outside no longer works, because CSRF protection is enabled ...
http://stackoverflow.com/a/29917946

Documentation: http://docs.spring.io/spring-security/site/docs/current/reference/html/csrf.html#when-to-use-csrf-protection

I would suggest the same as in the post ... enable it by hand (like you have done) in the login form and keep this line
```html
<form .... >
  ....other fields here....
  <input type="hidden"  name="${_csrf.parameterName}"   value="${_csrf.token}"/>
</form>
```